node/lib/cmd/cherry_pick.js

```diff
@@ -120,22 +119,25 @@ exports.executeableSubcommand = co.wrap(function *(args) {
         throw new UserError("No commit to cherry-pick.");
     }
 
-    const commitish = args.commit;
-    const annotated = yield GitUtil.resolveCommitish(repo, commitish);
-    if (null === annotated) {
-        throw new UserError(`\
+    // TOOD: check if we are mid-rebase already
+
+    const commits = args.commit;
+
+    const resolvedCommits = [];
+    for (let commitish of commits) {
+        const annotated = yield GitUtil.resolveCommitish(repo, commitish);
```

Comment:
to my understanding the multiple commits implies that the arg here is a "commitish" that should be resolved with git rev-list, not necessarily a list of commits, right? 

For example: `maint master..next` in vanilla git mean 
>all commits that are ancestors of maint or next, but not master or any of its ancestors.

Does `GitUtil.resolveCommitish` resolves commitish like git rev-list does?

Reply:
I think you're right -- that is the way that regular git cherry-pick does it.

That said, this is what the Two Sigma user requested, and it is an improvement that we could merge.  I would do the rev-list thing as a follow-up.